assets/queries/ansible/azure/redis_cache_allows_non_ssl_connections/metadata.json

```diff
@@ -2,8 +2,8 @@
   "id": "869e7fb4-30f0-4bdb-b360-ad548f337f2f",
   "queryName": "Redis Cache Allows Non SSL Connections",
   "severity": "MEDIUM",
-  "category": "Insecure Configurations",
-  "descriptionText": "Check if any Redis Cache resource allows non-SSL connections.",
+  "category": "Encryption",
+  "descriptionText": "Redis Cache resources should not allow non-SSL connections.",
```

Comment:
Please, update in `kics\assets\queries\terraform\azure\redis_cache_allows_non_ssl_connections\metadata.json`